moto/sagemaker/responses.py

```diff
@@ -239,3 +239,38 @@ def delete_training_job(self):
         training_job_name = self._get_param("TrainingJobName")
         self.sagemaker_backend.delete_training_job(training_job_name)
         return 200, {}, json.dumps("{}")
+
+    @amzn_request_id
+    def create_notebook_instance_lifecycle_config(self):
+        try:
+            lifecycle_configuration = self.sagemaker_backend.create_notebook_instance_lifecycle_config(
+                notebook_instance_lifecycle_config_name=self._get_param(
+                    "NotebookInstanceLifecycleConfigName"
+                ),
+                on_create=self._get_param("OnCreate"),
+                on_start=self._get_param("OnStart"),
+            )
+            response = {
+                "NotebookInstanceLifecycleConfigArn": lifecycle_configuration.notebook_instance_lifecycle_config_arn,
+            }
+            return 200, {}, json.dumps(response)
+        except AWSError as err:
+            return err.response()
```

Comment:
Is the try-except necessary? I don't see any errors being raised in models.py

Reply:
Ah, copy-paste from another operation.  But on further thought, there is an important exception I need to be throwing on create: already exists.  So I'm leaving the try/except in place here and adding code to detect duplicate name and raise in the backend method.